### DISCUSSION TITLE
Development mode in toolbar

### DISCUSSION THREAD

**Discussion by wtf403** [zen-browser/desktop#2928] [Ideas]
toolbar in Arc when you open page on localhost:
![image](https://github.com/user-attachments/assets/133bfe9d-7cc4-4bdb-ba15-b3d4fb06a81f)

**Comment by elyalon** [comment]
I tried making a zen mod for this and I believe it's not possible currently, because the current tab's URL is only accessible from the chrome's DOM by a text node (inside `#urlbar-input`), and it's not possible to create a CSS selector based on a text node, the current tab's URL needs to be exposed somewhere as an attribute (like `data-url="<url>"`)
